src/pages/Register/Business.tsx

```
@@ -0,0 +1,850 @@
+                        checked={isAgree}
+                        disabled={!allChecked}
+                        onChange={() => setIsAgree(!isAgree)}
+                        className="checkbox checkbox-accent mt-1"
+                      />
+                      <span className="text-sm text-gray-700">
+                        I agree to the{" "}
+                        <a
+                          className="font-semibold underline cursor-pointer text-[#4e9da8]"
+                          href="https://yagpsuctumdlmcazzeuv.supabase.co/storage/v1/object/public/website_documents/Terms%20of%20Service.pdf?t=2024-03-02T06%3A44%3A23.692Z"
+                          target="_blank"
+                        >
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)